.dockerignore

```diff
@@ -0,0 +1,8 @@
+# We do not need the git repository in docker images.
+.git/
+
+# We do not need the typical build directories in docker images.
+.idea/
+.build/
```

Comment:
Should `.build` and `_build` be replaced with `build-output` here?

`.dockerignore` doesn't include `build-output` right now.

Reply:
Fixed.